test/visual-diff/visual-tests

```diff
@@ -691,6 +692,19 @@
       "url": "examples/visual-tests/amp-autocomplete/amp-autocomplete.amp.html",
       "name": "amp-autocomplete",
       "interactive_tests": "examples/visual-tests/amp-autocomplete/amp-autocomplete.js",
+    },
+    {
+      "url": "examples/visual-tests/amp-ad/amp-ad.html",
+      "name": "AMP ad",
+      "enable_percy_javascript": true
```

Comment:
This doesn't seem necessary, as this allows JS to run on the Percy servers before creating the screenshot, but this doesn't include any AMP code (which gets removed before we upload the snapshot to Percy). Can you try removing this and seeing if this still works?

JavaScript code will still run inside the Travis VM for the page to be produced

Reply:
The screenshots without `enable_percy_javascript` is up, and the creative doesn't show up. I remember this also being the case when I tried locally a while ago; not sure what's the cause but I suspect it might be due to the iframes' contents not being visible in the static DOM shots.

Though to be sure can you restart the job?

Reply:
Hmm... one thing we do for tests that need to have a fixed window is to take their `outerHTML` and replace the entire page with an iframe, then set the iframe's `srcdoc` attribute to that previous `outerHTML`: https://github.com/ampproject/amphtml/blob/master/build-system/tasks/visual-diff/snippets/iframe-wrapper.js

Perhaps you can do something similar here - after the page is generated take the iframe's `innerHTML`, remove its `src` attribute, and set the previous `innerHTML` to its `srcdoc` value. You'll have to do it inside an `interactive_tests` file (which will have the downside of generating two screenshots - one without the custom code running and one with - this is not a big deal)

The downside of relying on `enable_percy_javascript` is that we have no guarantee for when the page "settles down", so this could be the cause of future flaky tests

Reply:
Done. Can you restart the run? Another diff test fails but it's probably unrelated.

Reply:
Looks really good now! Thanks for working through all this hassle :)

I restarted the job and it looks good. Approving the Percy build and merging this PR